cinnabar/git.py

```diff
@@ -114,7 +114,8 @@ def delete_ref(self, ref):
         self.update_ref(ref, '0' * 40)
 
     @classmethod
-    def config(self, name, remote=None, values={}, multiple=False):
+    def config(self, name, remote=None, values={}, multiple=False,
+               comma_separated_values=False):
```

Comment:
It should be possible to use `ConfigSetFunc` from `cinnabar.util` instead. That should simplify the handling of `all`.